setup.py

```diff
@@ -188,11 +222,12 @@ def setup_package():
                     cmdclass=cmdclass,
                     **extra_setuptools_args)
 
-    if (len(sys.argv) >= 2
-            and ('--help' in sys.argv[1:] or sys.argv[1]
-                 in ('--help-commands', 'egg_info', '--version', 'clean'))):
+    if (len(sys.argv) >= 2 and
+            ('--help' in sys.argv[1:] or sys.argv[1]
+            in ('--help-commands', 'egg_info', '--version', 'clean'))) \
```

Comment:
I would break after `in sys.argv[1:] ` this would make the code clearer and break in the tuple.